kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverterTest.java

```diff
@@ -736,6 +738,28 @@ public void testMapSchemalessConvertBytes() {
     assertEquals(((Map)(convertedMap.get("f3"))).get("f4"), Base64.getEncoder().encodeToString(helloWorld));
   }
 
+  @Test
```

Comment:
In case you'd accept the comment above, this could be the test:



```java
  @Test
  public void testVariableScaleDecimalConversion() {
    DebeziumLogicalConverters.VariableScaleDecimalConverter converter = new DebeziumLogicalConverters.VariableScaleDecimalConverter();

    assertEquals(LegacySQLTypeName.NUMERIC, converter.getBQSchemaType());

    try {
      converter.checkEncodingType(Schema.Type.STRUCT);
    } catch (Exception ex) {
      fail("Expected encoding type check to succeed.");
    }

    Struct specialValueDecimal = VariableScaleDecimal.fromLogical(VariableScaleDecimal.schema(), BigDecimal.ONE);
    Double convertedDecimal = converter.convert(specialValueDecimal);
    assertEquals(convertedDecimal, (Double) VariableScaleDecimal.toLogical(specialValueDecimal).toDouble());
  }
```